src/plugins/explorer/api/etherscan-compatible-api.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const axios = require('axios').default;
+
+class EtherscanCompatibleApi {
+  constructor({ baseURL }) {
+    this.api = axios.create({baseURL})
+  }
+
+  /**
+   * Returns a list of ERC20 Token Transfer Events hashes by Address
+   * @param {string} from start block
+   * @param {string} to end block
+   * @param {string} address wallet address
+   * @param {string} tokenAddress  address
+   * @returns {Promise<string[]>} array of transaction hashes
+   */
+  async getTokenTransactions(from, to, address, tokenAddress) {
+    const params = {
+      module: 'account',
+      action: 'tokentx',
+      sort: 'desc',
+      contractaddress: tokenAddress,
+      startBlock: from,
+      endBlock: to,
+      address,
+    }
+    const { data } = await this.api.get('', { params })
```

Comment:
Seems that both methods have the same code, but different params. Should we create a private method for reuse?